product/live_metrics/middleware_server.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# LiveMetrics configuration file
+#
+#   It defines the live metrics supported in a model that includes a LiveMetricsMixin.
+#   It maps the native id used in the provider for a metric with the column where this metric will be available in
+#   a LiveMetric virtual model.
+#
+#   Format:
+#   - Provider native metric id: MiQ metric id
+#
+- WildFly Memory Metrics~Heap Used: mw_heap_used
```

Comment:
Is there a need for such long string names for all metrics - does it serve some purpose?